### ISSUE TITLE
Network Switcher not available after restart of the Lisk Desktop

### ISSUE THREAD

**Issue by punkrock** [LiskHQ/lisk-desktop#2949]
v1.26.0, Windows 10

When opt-in "Network Switcher" in Settings it is opt-out again after a restart of the application.

**Comment by reyraa** [comment]
I don't see this problem. Please help me understand more about your issue.
Does this mean that the settings stored on your machine reset every time you open the app? Do other stored values such as bookmarks, dark/light mode, active token, etc remain as you set or those fall back to default values too?

**Comment by punkrock** [comment]
This is interesting. After I made a test bookmark settings are stored correctly. But with a fresh installation (no bookmarks), every time I start the app, I start from scratch (agree terms of use) and all settings I made (for example netwok switch on) are resetted. Now, after I did a bookmark, it works. Even if I delete the bookmark, settings are restored after restart of the app.

To confirm this issue i highly recommend to install Lisk Desktop on a fresh Windows 10 VM or something, without any registry entries. Then you start it for the first time, go to settings, turn on network switcher, close Lisk Desktop, restart it and see if settings are resetted to scratch.
